Aptoide/aptoide-client-v8
# [ASV-1020] Install placeholder hidden; Clicking on the app card now n…

**What does this PR do?**

This PR aims to fix small things to make the editorial card and view ready for production. The fixes done are:

- App card clicks outside of the button now navigate you to the appview for that app;

- Status bar no longer has white icons on white background;

- Hidden the placeholder install card while transition isn't ready;

- Mapping the info to the card in home was wrong;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] EditorialFragment.java

**How should this be manually tested?**

Check the ticket and the comment there and check the things mentioned both here and there;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1020](<https://aptoide.atlassian.net/browse/ASV-1020>)

**Code Review Checklist**
